src/Microsoft.Extensions.Configuration.AzureAppConfiguration/OfflineFileCache.cs

```diff
@@ -237,43 +194,23 @@ private async Task DoUpdate(string tempFile, int retry = 0)
             }
         }
 
-        private void EnsureOptions(AzureAppConfigurationOptions azconfigOptions)
+        private void CreateScopeToken(AzureAppConfigurationOptions azconfigOptions)
         {
             if (azconfigOptions == null)

```

Comment:
Can you move this to public API entry point and use debug.assert here. Also, param name should match public API.